CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.0.7
```

Comment:
We keep this under the title of 'Upcoming' in case we plan to roll out a major, minor or patch release.